requirements/base.txt

```diff
@@ -17,10 +17,10 @@ python-novaclient>=7.1.2,<16
 tenacity>=6.2,<8
 
 # for hastexo_guacamole_client
-django<=4.2.8
+django<=4.2.14
 channels<=4.0.0
 daphne<=4.0.0
-twisted<24 # drop this restriction once we drop Python 3.8 and 3.9 support
-mysqlclient<=2.2.1  # keep in sync with edx-platform
+twisted<24;python_version<="3.9" # drop this restriction once we drop Python 3.8 and 3.9 support
```

Comment:
Sorry, no, it doesn't work this way. This means "install `twisted`, using a version before 24.0.0, but only on Python releases earlier than 3.9, *and don't install `twisted` at all* for later releases." That's not what we want. Instead, we want:
```suggestion
twisted<24;python_version<"3.10"
twisted;python_version>="3.10"
```

Reply:
But don't we want for Python versions later than 3.9 to install it via Daphne/Channels as we did before? https://github.com/hastexo/hastexo-xblock/commit/eccd8a4b8b1fb8752f198508cb38e307c204e684

Reply:
Ah, sorry, so we do get `twisted`, just via `daphne` and not directly. OK, sounds good to me then. Thanks for the reminder. :)